README.md

```diff
@@ -255,13 +236,20 @@ func application(_ application: UIApplication, didFinishLaunchingWithOptions lau
 	        print("error = ", error)
 	    }
 
-	    // Enable or disable features based on the authorization status.
-	    // if you didn't register for remote notifications above you can call `registerForRemoteNotifications` here
-	    // Klaviyo SDK will automatically update the authorization status on next app launch
+        // you can check if the permission was granted or not but it's not necessary
```

Comment:
Nit:
- Fix indentations
- Comment about silent push seems inappropriate for a readme

Reply:
I realized the indentation and fix it but you are right about silent push. Removed it.